src/dotnet/commands/dotnet-test/VSTestArgumentConverter.cs

```diff
@@ -0,0 +1,169 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.DotNet.Cli
+{
+    using System.Collections.Generic;
+    using System.Linq;
+
+    /// <summary>
+    /// Converts the given arguments to vstest parsable arguments
+    /// </summary>
+    public class VSTestArgumentConverter
+    {
+        private const string verbosityString = "--logger:console;verbosity=";
+
+        private readonly Dictionary<string, string> ArgumentMapping = new Dictionary<string, string>
```

Comment:
FYI The converter seems doing many similar things like CLI parser. 

Use that might reduce most of the logic in this file.

https://github.com/dotnet/cli/blob/ba14323e5de7c1e3870d4064e9cbeff312edfe2d/src/dotnet/commands/dotnet-test/TestCommandParser.cs

Reply:
The thing with code is, it is not going to invoke msbuild eventually.
It directly calls into the vstest platform. If someone has already built the dlls and they want to use the test verb, we need to collate all the relevant args and send them to the platform, which dint seem possible using the parser.